storage/src/main/java/tech/pegasys/teku/storage/server/StorageConfiguration.java

```diff
@@ -275,7 +275,7 @@ public Builder blobsPruningLimit(final int blobsPruningLimit) {
     }
 
     public Builder retainedSlots(final long retainedSlots) {
-      if (retainedSlots < -1) {
```

Comment:
Are we missing a test case for this scenario? How come we changed the validation and no tests had to be updated?

Reply:
Added a test to ensure a negative value here is considered invalid